README.md

```diff
@@ -176,6 +174,25 @@ type Profile = {
 - `additionalLogoutParams`: dictionary of additional query params to add to 'logout' requests
 - `logoutCallbackUrl`: The value with which to populate the `Location` attribute in the `SingleLogoutService` elements in the generated service provider metadata.
 
+- **Extensions**
+- `extensions`: Optional, The extension provides a more flexible structure for expressing which combination of Attributes are requested by service providers in comparison to the existing mechanisms, [More about extensions](https://www.oasis-open.org/committees/download.php/55790/Connectis%20_protocol_extension_draft.pdf). It accept fully customize [XMLBuilder](https://www.npmjs.com/package/xmlbuilder) type. There are many possible value for `extensions` element. You can use [XMLBuilder](https://www.npmjs.com/package/xmlbuilder) to request attributes/data as per your need in SAML Request.
+
+```javascript
+// Example
+extensions: {
```

Comment:
I feel like the name `extensions` is too generic and might logically collide with something else. The documentation referenced continuously refers to "SAML Protocol Extension for Requesting Attributes per Request", so clearly they are leaving room for some other type of extension to do some other thing.

Would it make sense to call the section `samlExtensions` to make it more clear that these are extension that go directly into the SAML document and not some extension to `node-saml` or `passport-saml` functionality?

Reply:
agreed @cjbarth , `samlExtensions` is more clear. i'll make a change.